src/styles/utils/functions.scss

```diff
@@ -0,0 +1,14 @@
+@import "./variables.scss";
+
+@for $i from 28 through 31 {
+  .calendar--month-length-#{$i} :nth-child(n+#{$i + 1}) {
+    visibility: hidden;
+  }
+}
+
+@each $day, $i in $days {
+  .calendar--start-day-#{$day} :first-child {
+    margin-left: $each-cell-margin * $i;
+
+  }
+}
```

Comment:
The file is named functions.scss, but I couldn't locate any functions within it, only loops. Based on the styles, I believe it pertains to the BEM block calendar. It would be best to transfer it to the calendar.scss file.